content/2021-11-18-brigade-2-cuts-first-release-candidate.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Brigade 2 Cuts First Release Candidate"
+slug: "2021-11-18-brigade-2-cuts-first-release-candidate"
+description: "Brigade 2 has cut its first release candidate"
+date: "2021-11-18"
+author: "Kent Rancourt"
+summary: Brigade 2 has cut its first release candidate.
+---
+
+We have exciting news, Brigadiers!
+
+Yesterday afternoon, maintainers cut the first Brigade 2 release candidate. This release candidate (v2.0.0-rc.1) also marks Brigade 2's transition from serving the v2 beta API to serving the _stable_ v2 API. With this new level of stability, there's never been a better time to get started with Brigade 2, so please do consider heading over to our recently refreshed [QuickStart](https://v2--brigade-docs.netlify.app/intro/quickstart/) and taking us for a spin.
+
+The transition to the stable API means previous versions of gateways and other peripherals are not compatible with this release candidate, but as of this morning, we have _also_ transitioned every one of those projects to the stable API and cut a beta release of each. (All peripherals currently remain at a beta level of maturity despite Brigade 2 itself having advanced to a release candidate.)
+
+Barring any major surprises, we anticipate the only PRs merged between now and December 1 will be release-oriented chores, so we're targeting that date for our GA release -- just on time for the holiday season!
+
+Following the Brigade 2 GA release, maintainers will begin putting the finishing touches on all our peripherals and will release each of those individually as they reach maturity, with the GitHub gateway almost certainly being the first.
```

Comment:
nit: "peripherals" has been repeated a bunch of times already.

Reply:
Agree. There are three occurrences, which isn't _really_ a lot, but because it's a word we don't encounter a lot in daily use, seeing/hearing three times definitely does feel repetitive. I'll see if I can reword a little to avoid one use of that word.